content/en/02-installing-pixie/02-setting-up-k8s/05-k0s-setup.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "k0s"
+metaTitle: "Install | Install Guides | k0s"
+metaDescription: "How to install Pixie in k0s"
+order: 5
+featuredGuide: true
+icon: k0s-logo.svg
+---
+
+To set up Pixie on [k0s](https://kubernetes.io/docs/getting-started-guides/minikube/), you'll need to:
+
+1. Create a k0s cluster
+2. (Optional) Install a volume provisioner in the cluster
+3. Install Pixie
+
+## Create a k0s Cluster
+
+Follow the k0s [Quick Start Guide](https://docs.k0sproject.io/latest/install/) to set up a K8s cluster with a single node that includes both the controller and the worker:
+
+```bash
+# Download k0s
+curl -sSLf https://get.k0s.sh | sudo sh
+
+# Install k0s as a service
+sudo k0s install controller --single
+
+# Start k0s as a service
+sudo k0s start
+
+# Check service, logs and k0s status
+sudo k0s status
+```
+
+## (Optional) Install a Volumne Provisioner
+
+[Self-Hosted Pixie](/installing-pixie/install-guides/self-hosted-pixie) needs a PersistentVolume.
+
+1. Follow the [OpenEBS Guide](https://docs.openebs.io/docs/next/uglocalpv-hostpath.html) to set up a local PV.
+
+```bash
+export KUBECONFIG=path-to-kubeconfig-from-previous-step
```

Comment:
I would wrap the path-to-kubeconfig-from-previous-step in `<` and `>` or something, to make it very clear they are supposed to fill this out.